wsmaster/che-core-api-factory-github-common/src/main/java/org/eclipse/che/api/factory/server/github/AbstractGithubPersonalAccessTokenFetcher.java

```diff
@@ -37,6 +37,7 @@ public abstract class AbstractGithubPersonalAccessTokenFetcher
 
   private static final Logger LOG =
       LoggerFactory.getLogger(AbstractGithubPersonalAccessTokenFetcher.class);
+  private static final String OAUTH_PROVIDER_NAME = "github";
```

Comment:
It confuses having constant and constructor param with the same intent.
We need a clear comment explaining why we use "github" instead of provided provider name for certain cases.

Reply:
Added a javadoc.